help-wanted.md

```diff
@@ -0,0 +1,126 @@
+# Overview
+
+We use two labels `contributor/help-wanted` and `contributor/good-first-issue`
+to identify issues that have been specially groomed for both existing and new contributors. 
+The `contributor/good-first-issue` indicates that members have committed to providing extra assistance for new contributors.
+
+We also have some suggestions for using these labels to help grow and improve our community.
+
+## Help Wanted
+
+Items marked with the `contributor/help-wanted` label need to ensure that they are:
+
+- **Low Barrier to Entry**
+
+    It should be tractable for new contributors.
+    Documentation on how that type of change should be made should already exist.
+
+- **Clear Task**
+
+    The task is agreed upon and does not require further discussions in the community.
+    Call out if that area of code is untested and requires new fixtures.
+
+    API / CLI behavior is decided and included in the OP issue,
+    for example: *"The new command syntax is `foo bar NAME [--unFoo] [--timeout 5m]`"*,
+    with expected validations called out.
+
+- **Goldilocks priority**
+
+    Not too high that a core contributor should do it,
+    but not too low that it isn't useful enough for a core contributor to spend time to review it,
+    answer questions, help get it into a release, etc.
+
+- **Up-To-Date**
+
+    Often these issues become obsolete and have already been done,
+    are no longer desired, no longer make sense, have changed priority or difficulty, etc.
+    The maintainers of Traefik regularly groom and manage these issues to ensure they remain relevant.
+
+## Good First Issue
```

Comment:
In the same way, I would just add a message when we apply the label with something like "We will have special attention and help you with this PR/issue, you can find some doc here and here, don't hesitate to ask for help".
And avoid writing another external document to read.

I think we don't have to apply any process/LiveCycle that takes the number of `good-first-issue` PR into account to know if someone can work on a `contributor/help-wanted`. For me `help-wanted` and `good-first-issue` are not correlated and we can't know the skills nor the energy that the contributor could/would provide.

Reply:
> In the same way, I would just add a message when we apply the label with something like "We will have special attention and help you with this PR/issue, you can find some doc here and here, don't hesitate to ask for help".
And avoid writing another external document to read.

That's what the goal of the guidelines below are. That we prepare the issue accordingly to encourage new contributors and empower them with the details they need to start contributing. We should definitely timebox this effort, and be aware that not all these suggestions can always be met, but I would encourage the team to try and think about the bare minimum that is needed for new contributors to get rolling on an issue. (cf previous comment applies here as well)